### PR TITLE
updates to gap junction and intrinsic tables

### PR DESCRIPTION
**Pull request by scseeman**
associated changes for notebook/figure making

### CODE REVIEW

File: aisynphys/pipeline/multipatch/intrinsic.py [hunk 1]
```diff
@@ -89,9 +89,9 @@ def create_db_entries(cls, job, session):
             
             spike_features = lsa.mean_features_first_spike(analysis['spikes_set'])
             up_down = spike_features['upstroke_downstroke_ratio']
-            rheo = analysis['rheobase_i']
-            fi_slope = analysis['fi_fit_slope']
-            input_r = analysis['input_resistance']
+            rheo = analysis['rheobase_i'] * 1e9 #unscale from pA
```
Comment (campagnola) [inline aisynphys/pipeline/multipatch/intrinsic.py:92]: should be 1e-9 ?

File: aisynphys/ui/notebook.py [hunk 2]
```diff
@@ -148,13 +150,17 @@ def show_connectivity_matrix(ax, results, pre_cell_classes, post_cell_classes, c
...
-    distance_adjusted : bool
-        If True, use distance-adjusted connectivity metrics. See 
-        ``aisynphys.connectivity.measure_connectivity(sigma)``.
     cmap : matplotlib colormap instance
         The colormap used to generate colors for each matrix element
     norm : matplotlib normalize instance
         Normalize instance used to normalize connection probability values before color mapping
+    ctype: string
+        corresponds to dict key from ``aisynphys.connectivity.measure_connectivity``
+        distance_adjusted - use distance-adjusted connectivity metrics.
```
Comment (campagnola) [inline aisynphys/ui/notebook.py:159]: ctype and distance_adjusted are orthogonal--how would you do distance-adjusted gap junction connectivity?

File: aisynphys/ui/notebook.py [hunk 3]
```diff
@@ -233,6 +242,42 @@ def get_metric_data(metric, db, pre_classes=None, post_classes=None, pair_query_
     pairs_has_metric = pairs[~pairs[metric].isnull()]
     return pairs_has_metric, metric_name, units, scale, alpha, cmap, cmap_log, clim, cell_fmt
 
+def scatter_plots(metrics, db, pair_classes, pair_query_args, ax):
```
Comment (campagnola) [inline aisynphys/ui/notebook.py:245]: Would prefer a more descriptive name that `scatter_plots`, and documentation on the function arguments.

File: aisynphys/ui/notebook.py (hunk 2 above)
Comment (scseeman) [reply aisynphys/ui/notebook.py:159]: I guess I assumed the the distance-adjusted was not going to be broadly applicable like that, but we could split out into ctype='chemical' or 'electrical', and then have distance-adjusted go back to being a bool

File: aisynphys/ui/notebook.py [hunk 4]
```diff
@@ -173,11 +173,16 @@ def show_connectivity_matrix(ax, results, pre_cell_classes, post_cell_classes, c
...
-                found = result['n_gaps']
-            else:
+            if ctype == 'chemical':
                 found = result['n_connected']
+                if distance_adjusted:
+                    cp, cp_lower_ci, cp_upper_ci = result['adjusted_connectivity']
+                else:
+                    cp, cp_lower_ci, cp_upper_ci = result['connection_probability']
+            if ctype == 'electrical':
+                found = result['n_gaps']
```
Comment (campagnola) [inline aisynphys/ui/notebook.py:183]: check for `distance_adjusted` here and either return adjusted values or raise an exception saying this isn't implemented yet.

**Pull request merged**